api/src/shared/util/maps.util.ts

```diff
@@ -0,0 +1,8 @@
+export const mapToObject = (map: Map<string, any>): { [key: string]: any } =>
```

Comment:
why not replace this function with this?
```
const mapToObject = (map: Map<string, any>): { [key: string]: any } => 
  Object.fromEntries(map);
```

Not only it is more readable, it is also more efficient (since your original function uses the spread operator (...obj) inside a reduce callback, creating a new object on every iteration)

Reply:
Oh wow.. didn't realize we could create objects from Maps in that way.. #TIL Way better! 💯 

<img width="320" alt="image" src="https://github.com/user-attachments/assets/e72cc3c3-6944-4920-b81a-dbc80bb27e1d">